web/massastation/src/main.tsx

```diff
@@ -18,8 +18,9 @@ import mockServer from './mirage/server.js';
 
 const baseURL = import.meta.env.VITE_BASE_APP;
 
-if ([ENV.DEV, ENV.TEST].includes(baseURL)) {
-  mockServer(baseURL);
+// Add ENV.STANDALONE to the array to enable MirageJS
+if ([ENV.DEV, ENV.TEST].includes(import.meta.env.VITE_ENV)) {
+  mockServer(import.meta.env.VITE_ENV);
```

Comment:
Can we have something similar to the `baseURL` to use once down? baseENV or related...

```suggestion
if ([ENV.DEV, ENV.TEST].includes(baseENV)) {
  mockServer(baseENV);
```